src/App.css

```diff
@@ -1 +1,21 @@
 /* not empty */
+.pagination {
+  width: 20%;
+}
+
+.page-item{
+  padding-inline: 10px;
+  border: 1px solid grey;
+}
+
+.page-item.active {
+  background: blue;
+}
+
+.active .page-link{
```

Comment:
The selector `.active .page-link` might not work as intended if `.active` is meant to be a modifier of `.page-item`. If that's the case, it should be `.page-item.active .page-link` to increase specificity and ensure the styles are applied correctly.